### PR TITLE
Add __version__ endpoint

### PR DESCRIPTION
**Pull request by jasonthomas**
Fixes https://github.com/mozilla/addons-frontend/issues/321

r?

### CODE REVIEW

File: src/core/server/base.js [hunk 1]
```diff
@@ -65,6 +66,20 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
 
   app.use(Express.static(path.join(config.get('basePath'), 'dist')));
 
+  // Return version information as json
+  app.get('/__version__', function(req, res) {
+    fs.stat(version, (err, stats) => {
+      if (err) {
+        res.setHeader('Content-Type', 'text/plain');
+        res.sendStatus(404);
```
Comment (andymckay) [inline src/core/server/base.js]: How about 415? https://httpstatuses.com/415

File: src/core/server/base.js [hunk 2]
```diff
@@ -65,6 +66,20 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
 
   app.use(Express.static(path.join(config.get('basePath'), 'dist')));
 
+  // Return version information as json
+  app.get('/__version__', function(req, res) {
```
Comment (muffinresearch) [inline src/core/server/base.js]: Should be an arrow function.

File: src/core/server/base.js [hunk 3]
```diff
@@ -65,6 +66,20 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
...
+    fs.stat(version, (err, stats) => {
+      if (err) {
+        res.setHeader('Content-Type', 'text/plain');
+        res.sendStatus(404);
+       } else {
```
Comment (muffinresearch) [inline src/core/server/base.js]: Indents are off.

File: src/core/server/base.js [hunk 4]
```diff
@@ -65,6 +66,20 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
...
+        res.setHeader('Content-Type', 'application/json');
+        var stream = fs.createReadStream(version);
+        stream.pipe(res);
```
Comment (muffinresearch) [inline src/core/server/base.js]: You could just chain this on the previous line.

File: src/core/server/base.js [hunk 5]
```diff
@@ -65,6 +66,20 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
...
 
+  // Return version information as json
+  app.get('/__version__', function(req, res) {
+    fs.stat(version, (err, stats) => {
+      if (err) {
+        res.setHeader('Content-Type', 'text/plain');
+        res.sendStatus(404);
+       } else {
+        res.setHeader('Content-Type', 'application/json');
+        var stream = fs.createReadStream(version);
```
Comment (muffinresearch) [inline src/core/server/base.js]: Should be a `const`

**Comment by muffinresearch** [comment]
Few lint nits that I've mostly outlined. You can run `npm run lint` locally to check them.

**Comment by jasonthomas** [comment]
Thanks for the review. I've made the recommended changes.

**Comment by muffinresearch** [comment]
r+

**Pull request merged**